.pre-commit-config.yaml

```diff
@@ -59,3 +59,19 @@ repos:
     rev: v1.8.0
     hooks:
     -   id: rst-backticks
+- repo: https://github.com/nbQA-dev/nbQA
+  rev: 0.8.1
+  hooks:
+    - id: nbqa-pyupgrade
+      args: [--nbqa-mutate, --py36-plus]
+    - id: nbqa-black
+      args: [--nbqa-mutate]
+      language_version: python3
+    - id: nbqa-isort
+      args: [--nbqa-mutate]
+    - id: nbqa-flake8
+      args: [--extend-ignore=E402]
```

Comment:
Note: deactivating E402 (https://www.flake8rules.com/rules/E402.html) as it seems to clash with adding some line magics in cells.